pkg/kubelet/rkt/rkt.go

```diff
@@ -884,8 +925,10 @@ func (r *Runtime) preparePod(pod *api.Pod, pullSecrets []api.Secret) (string, *k
 	// TODO handle pod.Spec.HostPID
 	// TODO handle pod.Spec.HostIPC
 
+	markPodFinished := podFinishedMarkCommand(r.touchPath, r.runtimeHelper.GetPodDir(pod.UID), uuid)
```

Comment:
I think we can point out that for now we only have pod level finish time, but later we should support per app's finish time.